Modified language so it better adheres to IBM style guide (#66)
devfile/docs: docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

. Composite command
+

> You can define a composite command to chain multiple commands together. Reference the individual commands that are called from a composite command by the `name` of the command.  Specify a `parallel` boolean to determine if the commands within a composite command are being executed sequentially or in parallel.

Reviewer: * Instead of telling the user that they "can do something if they do something else", it's better to state the goal/target and then follow up with how to achieve it: "To achieve X, do Y." (I.e. be direct and authoritative.)
* In the second sentence, careful about meaning. When it's possible (but not required) to do something, tell the user how to do it, not to do it.
* The last sentence used "determine" incorrectly (in the orig. version, too). Devfiles say how things happen, so we don't use parameters to _determine_ the state of things. We use them to _specify_ how things are supposed to happen.

```suggestion
To chain multiple commands together, define a composite command. To reference the individual commands that are called from a composite command, use the `name` of the command. To specify whether commands within a composite command are to be executed sequentially or in parallel, define the `parallel` boolean.
```
